langchain/document_loaders/readthedocs.py

```diff
@@ -12,7 +12,6 @@ class ReadTheDocsLoader(BaseLoader):
     def __init__(
         self,
         path: str,
-        encoding: Optional[str] = None,
```

Comment:
What's the motivation for removing the ability to select the encoding? It may make sense to default to utf-8, but I don't think it makes sense to prevent someone from decoding in utf-16 or ascii